src/modules/SaveSelector.ts

```diff
@@ -29,7 +29,7 @@ export default class SaveSelector {
             $('#saveSelectorContextMenu').html(`
                 <a class="dropdown-item bg-success" href="#" onclick="Save.key = '${key}'; SaveSelector.Download('${key}')">Download (backup)</a>
                 <a class="dropdown-item bg-info" href="#" onclick="Save.key = '${key}'; document.querySelector('#saveSelector').remove(); App.start();">Load</a>
-                <a class="dropdown-item bg-warning" href="#"><label class="clickable my-0" for="import-save" onclick="Save.key = '${key}';">Import (overwrite)</label></a>
+                <a class="dropdown-item bg-warning p-0" href="#"><label class="clickable my-0" style="padding:.75rem 1.5rem;" for="import-save" onclick="Save.key = '${key}';">Import (overwrite)</label></a>
```

Comment:
```suggestion
                <a class="dropdown-item bg-warning p-0" href="#"><label class="clickable my-0 w-100" style="padding:.75rem 1.5rem;" for="import-save" onclick="Save.key = '${key}';">Import (overwrite)</label></a>
```
The label doesn't fill the whole width of the dropdown item by itself, it requires some encouragement.